src/app/bulk-edit/bulk-edit-select/bulk-edit-select.component.ts

```diff
@@ -42,6 +55,8 @@ export class BulkEditSelectComponent implements OnInit {
   availableProfiles: ProfileSummary[];
   setupForm: FormGroup;
 
+  private unsubscribe$ = new Subject();
```

Comment:
can you explain this to me if you have time, its seems useful and I think i get whats its doing but I would like to know how its working.